<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateSingleDoiXmlDialog</class>
 <widget class="QDialog" name="CreateSingleDoiXmlDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>555</width>
    <height>643</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Create DOI XML file</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout_5">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox_4">
     <property name="title">
      <string>Output directory</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLineEdit" name="DirLineEdit">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="browseDir_pushButton">
          <property name="text">
           <string>Browse...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>DOI</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>10.2312/</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="DOI_lineEdit"/>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="label_19">
        <property name="text">
         <string>auto: journal.year.volume.first page</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <spacer name="horizontalSpacer_5">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>137</width>
          <height>17</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Article</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Author(s):</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="5">
       <widget class="QLineEdit" name="Authors_lineEdit"/>
      </item>
      <item row="1" column="1" colspan="3">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Surname1, Firstname1; Surname2, Firstname2; ...</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Title:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1" colspan="5">
       <widget class="QLineEdit" name="Title_lineEdit"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Date:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="Date_lineEdit">
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Volume:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="3">
       <widget class="QLineEdit" name="Volume_lineEdit">
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="3" column="4">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Issue:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="5">
       <widget class="QLineEdit" name="Issue_lineEdit">
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>First page:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="FirstPage_lineEdit">
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Last page:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="3">
       <widget class="QLineEdit" name="LastPage_lineEdit">
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="4" column="4">
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>Language:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="5">
       <widget class="QComboBox" name="Language_comboBox">
        <item>
         <property name="text">
          <string>de</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>en</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>es</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>fr</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>ru</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>Description:</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1" colspan="5">
       <widget class="QTextEdit" name="Description_textEdit">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>100</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="label_12">
        <property name="text">
         <string>Discipline:</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1" colspan="2">
       <widget class="QComboBox" name="Discipline_comboBox">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>0</height>
         </size>
        </property>
        <item>
         <property name="text">
          <string>500 Science</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>510 Mathematics</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>520 Astronomy</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>530 Physics</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>540 Chemistry</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>550 Earth sciences and geology</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>560 Fossils and prehistoric life</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>570 Life sciences; biology</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>580 Plants (Botany)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>590 Animals (Zoology)</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="label_13">
        <property name="text">
         <string>Type:</string>
        </property>
       </widget>
      </item>
      <item row="7" column="1" colspan="2">
       <widget class="QComboBox" name="Type_comboBox">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>0</height>
         </size>
        </property>
        <item>
         <property name="text">
          <string>Book chapter</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Book preface</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Conference abstract</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Conference paper</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Conference poster</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Conference talk</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Journal article</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Journal comment</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Journal letter</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Journal review</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Lecture notes</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Newspaper article</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Report</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Report chapter</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Journal</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QLabel" name="label_14">
        <property name="text">
         <string>Title:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="2">
       <widget class="QLineEdit" name="ProceedingsTitle_lineEdit"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_15">
        <property name="text">
         <string>Publisher:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" colspan="2">
       <widget class="QLineEdit" name="ProceedingsPublisher_lineEdit"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_16">
        <property name="text">
         <string>Contributor:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1" colspan="2">
       <widget class="QLineEdit" name="ProceedingsContributor_lineEdit"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_17">
        <property name="text">
         <string>Place:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1" colspan="2">
       <widget class="QLineEdit" name="PublicationPlace_lineEdit"/>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_18">
        <property name="text">
         <string>Identifier</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="ProceedingsIdentifier_lineEdit">
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>243</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="New_pushButton">
       <property name="text">
        <string>New</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Example_pushButton">
       <property name="text">
        <string>Example</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="OK_pushButton">
       <property name="text">
        <string>&amp;OK</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Cancel_pushButton">
       <property name="text">
        <string>&amp;Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>DirLineEdit</tabstop>
  <tabstop>browseDir_pushButton</tabstop>
  <tabstop>DOI_lineEdit</tabstop>
  <tabstop>Authors_lineEdit</tabstop>
  <tabstop>Title_lineEdit</tabstop>
  <tabstop>Date_lineEdit</tabstop>
  <tabstop>Volume_lineEdit</tabstop>
  <tabstop>Issue_lineEdit</tabstop>
  <tabstop>FirstPage_lineEdit</tabstop>
  <tabstop>LastPage_lineEdit</tabstop>
  <tabstop>Language_comboBox</tabstop>
  <tabstop>Description_textEdit</tabstop>
  <tabstop>Discipline_comboBox</tabstop>
  <tabstop>Type_comboBox</tabstop>
  <tabstop>ProceedingsTitle_lineEdit</tabstop>
  <tabstop>ProceedingsPublisher_lineEdit</tabstop>
  <tabstop>ProceedingsContributor_lineEdit</tabstop>
  <tabstop>PublicationPlace_lineEdit</tabstop>
  <tabstop>ProceedingsIdentifier_lineEdit</tabstop>
  <tabstop>New_pushButton</tabstop>
  <tabstop>Example_pushButton</tabstop>
  <tabstop>OK_pushButton</tabstop>
  <tabstop>Cancel_pushButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
